website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,298 @@
+part of the [Manager][ctrl-manager] scope.
+
+**IMPORTANT:** Note that this is not intended to be used for excluding namespaces, this is better done via a Predicate. 
+Also note that you may face performance issues when using this with a high number of namespaces.
+
+## Restricting Roles and permissions
+
+The operator-scope defines the [Manager][ctrl-manager] cache scope but not the permissions to access the resources. After updating the Manager's scope to be Namespaced, the [Kubernetes Role-Based Access Control or the (RBAC)][k8s-rbac] permissions should be restricted accordingly.
+
+In the projects, the permissions are specified in the directory `config/rbac/`. The RBAC manifest at `config/rbac/role.yaml` and `config/rbac/role_binding.yaml` specifies the permissions needed by the operator to manage its resources, and is a `ClusterRole` by default. 
+ 
+**NOTE** The files `auth_proxy_client_clusterrole.yaml`, `auth_proxy_role.yaml`, `auth_proxy_role_binding.yaml` are also 
+defined to allow permissions across the cluster as well and are used by the operator. More info see: 
+[kube-rbac-proxy][kube-rbac-proxy]. Also, it is important to highlight that it is an alpha feature that is not officially 
+supported and can be disabled by commenting the lines in the config `config/rbac/kustomization.yaml` and in the 
+`config/default/kustomization.yaml`. See:
+
+-  In `config/rbac/kustomization.yaml`:
+```yaml
+...
+# Comment the following 4 lines if you want to disable
+# the auth proxy (https://github.com/brancz/kube-rbac-proxy)
+# which protects your /metrics endpoint.
+- auth_proxy_service.yaml
+- auth_proxy_role.yaml
+- auth_proxy_role_binding.yaml
+- auth_proxy_client_clusterrole.yaml
+```
+
+- In `config/default/kustomization.yaml`:
+```yaml
+...
+patchesStrategicMerge:
+  # Protect the /metrics endpoint by putting it behind auth.
+  # If you want your controller-manager to expose the /metrics
+  # endpoint w/o any authn/z, please comment the following line.
+- manager_auth_proxy_patch.yaml
+...
+```
+
+And then, see that for each API (CRD) scaffolded a specific role file with permissions to `edit` or `view` the Kind 
+resource in the cluster is created too. It is not used by the project, but are helpers to allow you to perform specific 
+configurations. For example, after creating a new API(CRD) with the Memcached Kind, we will able to see the roles 
+`memcached_editor_role.ymal` and `memcached_viewer_role.yaml` created there.
+
+### Changing the permissions 
+
+To change the scope of the RBAC permissions from cluster-wide to a specific namespace, you will need to use a `Role` 
+instead of `ClusterRole` and `RoleBinding` instead of `ClusterRoleBinding`. Note that, the project uses `markers` to 
+define the properties of these RBAC files which are generated. The [RBAC ClusterRole][rbac-clusterrole] are created by 
+default because of the `markers` in the Kind's controllers are:
```

Comment:
- Remove extra `,`.
- Link to [RBAC markers](https://book.kubebuilder.io/reference/markers/rbac.html) since we're mentioning them here for the first time.
- Highlight why default is ClusterRole.
- Also rephrase since markers don't "define the properties of all files". Just the `role.yaml` manifest.
```suggestion
To change the scope of the RBAC permissions from cluster-wide to a specific namespace, you will need to use a `Role` 
instead of `ClusterRole` and `RoleBinding` instead of `ClusterRoleBinding`. 

[`RBAC markers`][rbac-markers] defined in the controller (e.g `controllers/cache/memcached_controller.go`)
are used to generate the operator's `Role`/`ClusterRole` manifest `config/rbac/role.yaml`.
The default markers don't specify a `namespace` property and will result in a ClusterRole.

[rbac-markers]: https://book.kubebuilder.io/reference/markers/rbac.html
```

Reply:
Addressed with caveats.